src/components/OptionsPage/Checkbox/LabeledCheckbox.tsx

```diff
@@ -48,7 +55,18 @@ export const LabeledCheckbox: FC<{
         value={props.value}
       />
 
-      <Paragraph>{props.description}</Paragraph>
+      <Paragraph>
+        {props.learnMoreLink === undefined ? (
+          props.description
+        ) : (
+          <>
+            {props.description}{' '}
+            <a className={paragraph} href={props.learnMoreLink} rel="noreferrer noopener" target="_blank">
+              {t('optionsPage_optionControl_learnMoreLink')}
+            </a>
```

Comment:
`className={paragraph}` attribute should not be used here because it may produce unexpected styling issues in the future. As you can see, the selector is [declared as `.paragraph a`](https://github.com/camellia-app/camellia/blob/b852024de03d6833e46a14d251ea520a1ae2f934/src/components/OptionsPage/Paragraph/Paragraph.module.css#L7-L12) which means some parent element has class `paragraph` and it has an `<a>` element _inside_.